src/lib/isInDrawer.js

```diff
@@ -0,0 +1,4 @@
+export default function isInDrawer() {
+  const urlParams = new URLSearchParams(window.location.search)
```

Comment:
`URLSearchParams` supposed to be supported on most modern browsers, except for IE. I wonder if this gets transpiled?